czech/member.lang.php

```diff
@@ -196,6 +196,8 @@
 $l['already_logged_in'] = "Poznámka: již jste přihlášen jako {1}.";
 $l['admin_edit_in_acp'] = "Upravit tohoto uživatele v Administraci";
 $l['admin_ban_in_acp'] = "Banovat tohoto uživatele v Administraci";
+$l['admin_lift_ban_in_acp'] = "Zrušit ban v administraci";
```

Comment:
```suggestion
$l['admin_lift_ban_in_acp'] = "Zrušit ban v Administraci";
```

Když už máme všude velké A, tak i tady musí být capital A.